jetty-websocket/javax-websocket-server/src/main/java/org/eclipse/jetty/websocket/javax/server/config/ContainerDefaultConfigurator.java

```diff
@@ -73,7 +72,9 @@ public boolean checkOrigin(String originHeaderValue)
         }
         catch (Exception e)
         {
-            throw new InstantiationException(String.format("%s: %s", e.getClass().getName(), e.getMessage()));
+            InstantiationException instantiationException = new InstantiationException(String.format("%s: %s", e.getClass().getName(), e.getMessage()));
```

Comment:
No need to pass String information: now that you call `initCause()` that will be available.